docs/install-machine.md

```diff
@@ -57,6 +57,21 @@ to your PATH.
         $ docker-machine -v
         machine version 0.5.0 (3e06852)
 
+## Installing bash completion scripts
+
+There are a [number of scripts](../contrib/completion/bash) which add command
```

Comment:
@nathanleclaire The link work when viewed [on my fork](https://github.com/leedm777/machine/blob/completion/docs/install-machine.md). I'll gladly take recommendations if there's a better way to link to the scripts.